common/djangoapps/entitlements/tests/test_entitlements_api.py

```diff
@@ -0,0 +1,162 @@
+import json
+import unittest
+import uuid
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from entitlements.tests.factories import CourseEntitlementFactory
+from entitlements.models import CourseEntitlement
+from student.tests.factories import CourseEnrollmentFactory, UserFactory, TEST_PASSWORD
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class EntitlementsTest(ModuleStoreTestCase):
+    """
+    Entitlements API/View Tests
```

Comment:
This docstring doesn't tell us anything we don't already know from the file name/location.